app/controllers/concerns/current_cart.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Cart
+module CurrentCart
```

Comment:
przenioslbym to do application_controller

Reply:
Done.